packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -105,11 +99,31 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
       ],
       className
     ],
+    input: [
+      {
+        position: 'absolute',
+        background: 'none',
+
+        opacity: 0,
+        selectors: {
+          [`.${IsFocusVisibleClassName} &:focus + label::before`]: {
+            outline: '1px solid ' + theme.palette.neutralSecondary,
+            zIndex: ZIndexes.FocusStyle,
+            outlineOffset: '2px',
+            selectors: {
+              [HighContrastSelector]: {
+                outline: '1px solid ActiveBorder'

```

Comment:
System colors are deprecated in CSS 3 http://www.w3.org/TR/css-color-3/ though still seem to work. 
Not a blocking issue here but may be good to check with James Jackson for guidelines for high contrast colors

Reply:
I was primarily following existing art where we used System colors extensively for MS Edge high contrast (for the various Fabric components). 

@jspurlin do you have a recommendation here?

I can also check with James Jackson. Thanks for the pointer Amy.

Reply:
This is the first I've heard of system colors being deprecated. Without leveraging those values, we cannot correctly style complex/custom controls in HC (in IE and Edge). I don't see support for these going away any time soon and recommend that we continue to use then due to the fact that there are no other alternative options.

---
In reply to: [245792511](https://github.com/OfficeDev/office-ui-fabric-react/pull/7521#discussion_r245792511) [](ancestors = 245792511)

Reply:
I'm with @jspurlin here. If people customize their high contrast slots then we don't honor them without it. FYI, I've ok'd the screener tests.

Reply:
Thanks for input @jspurlin and @betrue-final-final , I'll skip that email to James Jackson then.

Reply:
Kicked off a thread with Edge on what the future is here... I'm surprised about the deprecation.

Reply:
Resolution is to use these system colors (deprecated, but are still the recommended way to support Windows high contrast).